flink-runtime/src/main/java/org/apache/flink/runtime/security/token/NoOpDelegationTokenManager.java

```diff
@@ -35,12 +34,12 @@ public NoOpDelegationTokenManager() {
     }
 
     @Override
-    public void obtainDelegationTokens(Credentials credentials) {
+    public void obtainDelegationTokens(DelegationTokenContainer container) {
         LOG.debug("obtainDelegationTokens");
     }
 
     @Override
-    public void start(DelegationTokenListener delegationTokenListener) {
+    public void start(Listener listener) {
```

Comment:
Do we still need the `NoOpDelegationTokenManager`?

Reply:
I think yes, because null checks cost more in code than a noop manager.